### PR TITLE
HDDS-6790: Add document for Merge RocksDB in datanode feature.

### PR DESCRIPTION
**Pull request by ChenSammi**
https://issues.apache.org/jira/browse/HDDS-6790

### CODE REVIEW

**Comment by ChenSammi** [comment]
cc @guihecheng

File: hadoop-hdds/docs/content/feature/dn-merge-rocksdb.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+ the data volume configured in "hdds.datanode.dir". 
+
+For some advanced cluster admins who have the high
+  performance requirement, he/she can leverage quick storages to save RocksDB
+  . In this case, configure these two properties.  
+
+```XML
+<property>
+   <name>hdds.datanode.container.db.dir</name>
+   </value>
```
Comment (guihecheng) [inline hadoop-hdds/docs/content/feature/dn-merge-rocksdb.md:67]: Seems to be a misformat.

**Review by guihecheng — COMMENTED** [review]
Mostly LGTM.

File: hadoop-hdds/docs/content/feature/dn-merge-rocksdb.md (hunk 1 above)
Comment (ChenSammi) [reply hadoop-hdds/docs/content/feature/dn-merge-rocksdb.md]: Right.

**Review by guihecheng — APPROVED** [review]
LGTM+1

**Pull request merged**